### DISCUSSION TITLE
MaaTouch没有加入性能测试是有什么缘由吗？

### DISCUSSION THREAD

**Discussion by LostMyMind** [LmeSzinc/AzurLaneAutoScript#2594] [General]
Maatouch似乎没有加入性能测试下的模拟器选项里，而且控制方案里也没有MaaTouch的介绍，是有什么缘由吗？还是因为忘了（

**Comment by LmeSzinc** [comment]
因为和 minitouch 一样都是 0ms 延迟，没必要单独测试。minitouch 服务由 atx-agent 守护可以直接连接，而 maatouch 需要每次都启动，所以一般用 minitouch。
